contrib/convert_twitter_archive_to_json_data_source.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""This file converts your Twitter Archive data file into a simple JSON file like this:
+
+{
+    "2019-01-01": 123,
+    "2019-01-02": 456,
+    ...
+}
+
+The resulting file serves as an input file to the GitHubPoster JSON data source.
+
+More on your Twitter Archive: https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive
+
+More on GitHubPoster project: https://github.com/wzyboy/GitHubPoster
+"""
+
+import json
+import argparse
+from datetime import datetime
+from collections import Counter
+
+
+# Wed Jul 08 15:04:01 +0000 2009
+TIMESTAMP_FMT = '%a %b %d %H:%M:%S %z %Y'
+
+
+def convert(input_file, output_file):
+
+    with open(input_file, 'r') as f:
+        content = f.read()
+
+    # Read JSON payload from the JavaScript file
+    preamble = 'window.YTD.tweet.part0 = '
+    content = content[len(preamble):]
+
+    counter = Counter()
```

Comment:
IMO `defaultdict(int)` maybe OK here? no need `Counter`

Reply:
`collections.Counter` is also a subclass of `dict` like `defaultdict`, so it should make no difference, performance-wise. Yet a `Counter` provides more readability as the name implies that it counts stuff.

Reply:
Cool, got your point.